package.json

```diff
@@ -8,10 +8,10 @@
     "lint:fix": "eslint src --fix --ext .js,.jsx,.ts,.tsx",
     "start": "npm run clean && npm run cp:all && webpack-dev-server --open",
     "start:qa": "npm run clean && npm run cp:all && webpack-dev-server --node-env qa --open",
-    "start:prod": "npm run clean && npm run cp:all && webpack-dev-server --node-env production --open",
+    "start:prod": "cross-env npm run clean && npm run cp:all && webpack-dev-server --open",
     "build": "webpack",
     "build:qa": "webpack --node-env qa",
-    "build:prod": "webpack --node-env production",
+    "build:prod": "cross-env webpack",
```

Comment:
You have changed build:prod, but not build:qa or Build - was this intentional?

Reply:
yes because this issue there is just in production,